scripts/generate-package-table-for-readme.js

```diff
@@ -0,0 +1,120 @@
+  ['libp2p/js-libp2p-mplex', 'libp2p-mplex'],
+  ['libp2p/js-libp2p-railing', 'libp2p-railing'],
+  ['libp2p/js-libp2p-secio', 'libp2p-secio'],
+  ['libp2p/js-libp2p-tcp', 'libp2p-tcp'],
+  ['libp2p/js-libp2p-webrtc-star', 'libp2p-webrtc-star'],
+  ['libp2p/js-libp2p-websocket-star', 'libp2p-websocket-star'],
+  ['libp2p/js-libp2p-websockets', 'libp2p-websockets'],
+
+  'Data Types',
+  ['ipfs/js-ipfs-block', 'ipfs-block'],
+  ['ipfs/js-ipfs-unixfs', 'ipfs-unixfs'],
+  ['libp2p/js-peer-id', 'peer-id'],
+  ['libp2p/js-peer-info', 'peer-info'],
+  ['multiformats/js-multiaddr', 'multiaddr'],
+  ['multiformats/js-multihash', 'multihashes'],
+
+  'Generics/Utils',
+  ['ipfs/js-ipfs-api', 'ipfs-api'],
+  ['xicombd/ipfs-multipart', 'ipfs-multipart'],
+  ['ipfs/is-ipfs', 'is-ipfs'],
+  ['multiformats/js-multihashing', 'multihashing'],
+  ['multiformats/js-mafmt', 'mafmt'],
+
+  'Crypto',
+  ['libp2p/js-libp2p-crypto', 'libp2p-crypto'],
+  ['libp2p/js-libp2p-keychain', 'libp2p-keychain']
+]
+
+const isItemPackage = (item) => {
+  return Array.isArray(item)
+}
+
+const packageBadges = [
+  // Package
+  (gh, npm) => `[\`${npm}\`](//github.com/${gh})`,
+  // Version
+  (gh, npm) => `[![npm](https://img.shields.io/npm/v/${npm}.svg?maxAge=86400&style=flat-square)](//github.com/${gh}/releases)`,
+  // Deps
+  (gh, npm) => `[![Dep](https://david-dm.org/${gh}.svg?style=flat-square)](https://david-dm.org/${gh})`,
+  // DevDeps
+  (gh, npm) => `[![Dep](https://david-dm.org/${gh}.svg?style=flat-square)](https://david-dm.org/${gh})`,
+  // CI
+  (gh, npm) => {
+    // Need to fix the path for jenkins links, as jenkins adds `/job/` between everything
+    const jenkinsPath = gh.split('/').join('/job/')
+    return `[![Build Status](https://ci.ipfs.team/buildStatus/icon?job=${gh}/master)](https://ci.ipfs.team/job/${jenkinsPath}/job/master/)`
+  },
+  // Coverage
+  (gh, npm) => `[![Coverage Status](https://coveralls.io/repos/github/${gh}/badge.svg?branch=master)](https://coveralls.io/github/${gh}?branch=master)`
+  // TODO once we're ready for codecov badge instead of coveralls
+  // (gh, npm) => `[![Coverage Status](https://codecov.io/gh/${gh}/branch/master/graph/badge.svg)](https://codecov.io/gh/${gh})`
+]
+
+// Creates the table row for a package
+const generatePackageRow = (item) => {
+  const row = packageBadges.map((func) => {
+    // First string is GitHub path, second is npm package name
+    return func(item[0], item[1])
+  }).join(' | ')
+  const fullRow = `| ${row} |`
+  return fullRow
+}
+
+// Generates a row for the table, depending if it's a package or a heading
+const generateRow = (item) => {
+  if (isItemPackage(item)) {
+    return generatePackageRow(item)
+  } else {
+    return `| **${item}** |`
+  }
+}
+
+const header = `| ${columns.join(' | ')} |`
+const hr = `| ${columns.map(() => '---------').join('|')} |`
+
+const toPrint = [
+  header,
+  hr,
+  rows.map((row) => generateRow(row)).join('\n')
+]
+
+toPrint.forEach((t) => console.log(t))
```

Comment:
@VictorBjelkholm can you make this be a module that we can just feed it some JSON? It will get used by IPLD, and libp2p

Reply:
Let's put that as a future task, the scope of this PR is to remove CircleCI

Reply:
Fine™